app/templates/views/email-link-interstitial.html

```diff
@@ -3,23 +3,19 @@
 {% from "components/page-footer.html" import page_footer %}
 
 {% block per_page_title %}
-  Sign in
+    Click below to complete email re-verification and finish signing in.
 {% endblock %}
 
 {% block maincolumn_content %}
 
-<div class="js-hidden">
+<div>
 
-  {{ page_header('Sign in') }}
+    <h1 class="font-body-2xl margin-bottom-3">Click below to complete email re-verification and finish signing in.</h1>
```

Comment:
Ah, did you add these classes to the `h1` element?  Maybe there are others we can choose from to adjust the formatting further if need be!

Reply:
I saw that class was added in a recent PR (after a bunch of front end changes were pulled in) for h1's. I think that's the structure we are going for.